spec/models/document_spec.rb

```diff
@@ -3,49 +3,72 @@
 require 'rails_helper'
 
 describe Document do
+  subject(:document) { described_class.new(valid_params) }
+
   let(:valid_params) do
     {
-      id: 'a123',
+      id: 'b246',
       language: 'en',
-      path: 'http://www.agency.gov/page1.html',
-      title: 'My Title',
-      created: DateTime.new(2020, 1, 1),
-      changed: DateTime.new(2020, 1, 2),
-      description: 'My Description',
-      content: 'some content',
+      path: 'http://www.document_agency.gov/page1.html',
+      title: 'Title of Document',
+      created: DateTime.new(2022, 2, 2),
+      changed: DateTime.new(2022, 2, 4),
+      description: 'Description of Document',
+      content: 'content document content',
+      mime_type: 'text/plain',
       promote: true,
-      tags: 'this,that',
-      click_count: 5
+      tags: 'argle_tag,bargle_tag',
+      click_count: 6
     }
   end
 
   describe 'attributes' do
-    subject(:document) { described_class.new(valid_params) }
-
     it do
       is_expected.to have_attributes(
-        id: 'a123',
-        language: 'en',
-        path: 'http://www.agency.gov/page1.html',
-        title: 'My Title',
-        created: DateTime.new(2020, 1, 1),
-        changed: DateTime.new(2020, 1, 2),
-        description: 'My Description',
-        content: 'some content',
+        click_count: 6,
+        tags: 'argle_tag,bargle_tag',
         promote: true,
-        tags: 'this,that',
-        click_count: 5
+        mime_type: 'text/plain',
+        content: 'content document content',
+        description: 'Description of Document',
+        changed: DateTime.new(2022, 2, 4),
+        created: DateTime.new(2022, 2, 2),
+        title: 'Title of Document',
+        path: 'http://www.document_agency.gov/page1.html',
+        language: 'en',
+        id: 'b246'
       )
     end
 
     it 'sets default timestamps' do
       expect(document.created_at).to be_a Time
       expect(document.updated_at).to be_a Time
     end
+
+    context 'with the minimum required params' do
+      subject(:document) do
+        described_class.new(
+          language: 'en',
+          path: 'https://foo.gov'
+        )
+
+        it { is_expected.to be_valid }
+      end
+    end
+  end
+
+  describe 'invalid?' do
+    subject(:document) { described_class.new(valid_params.merge(mime_type: 'text/not_a_valid_mime_type')) }
+
+    it 'returns true and sets an error message for invalid MIME type' do
+      expect(document.invalid?).to be true
+      expect(document.errors.messages[:mime_type]).to include 'is invalid'
+    end
   end
```

Comment:
See my original comment (https://github.com/GSA/i14y/pull/200#discussion_r946938115) and the example spec I wrote (https://github.com/GSA/i14y/blob/2ba90eb0ff302e460dd5e7531cd2aeb272aa87d5/spec/models/document_spec.rb#L53). 

Please:
- move this into the `validations` block
- use a `context` rather than a `describe` block
- break the `subject` declaration into several lines for readability (the general rule of thumb is that lines should be no more than 80-100 char)

The error message spec should be separate from the validation spec (which can be a one-liner). We break this rule on occasion, but there's no need to combine them here. See https://www.betterspecs.org/#single